backend/kaggle/datasets_manager.py

```diff
@@ -0,0 +1,108 @@
+import os
+import csv
+from os import path
+from pathlib import Path
+
+DEFAULT_DOWNLOAD_PATH = path.join(Path.home(), Path('.kaggle/'))
+
+
+def download(dataset, download_path=DEFAULT_DOWNLOAD_PATH):
+    '''
+    Parameters:
+        dataset : a string.
+        download_path : a string or any Path object.
+
+    Abilities:
+        To download a dataset from kaggle then save it to a directory.
+
+    Returns:
+        dataset_path : full folder path where dataset is downloaded.
+        None : if dataset is already existed in download path.
+
+    Example:
+        This is a link to a kaggle dataset:
+        https://www.kaggle.com/aungpyaeap/tictactoe-endgame-dataset-uci
+
+        dataset = 'aungpyaeap/tictactoe-endgame-dataset-uci'
+        download_path = '~/.kaggle/'
+        dataset_path = download(dataset, download_path)
+
+        dataset_path = '~/.kaggle/aungpyaeap/tictactoe-endgame-dataset-uci/'
+    '''
+    try:
+        os.mkdir(path.join(download_path, Path(dataset.split('/')[0])))
+        dataset_path = path.join(download_path, Path(dataset))
+        os.mkdir(dataset_path)
+
+    except FileExistsError:
+        return None
+
+    os.system(f'kaggle datasets download {dataset} -p {dataset_path} --unzip')
+    return dataset_path
+
+
+class skin_cancer_mnist:
+    def __init__(self, dataset_path, full_path=False):
+        '''
+        Parameters:
+            dataset_path : a string or any Path object.
+            full_path : if True then is non-relative path.
+
+        Abilities:
+            To make an object manipulating dataset skin-cancer-minist-ham10000
+            from kmader.
+
+        Returns:
+            A skin_cancer_mnist object has
+                attributes:
+                    __dataset_url : a string
+                    __dataset_path : a Path object
+                    __metadata_path : a Path object
+                    __img_metadata_list : a list of tuples.
+                        Each tuple has two elements.
+                        First element (Path object) is a path to an image
+                        Second element (Dict object) is a metadata of an image
+                functions
+                    get_img_metadata_list()
```

Comment:
The `__init__` is an initializer and they don't return anything. If you're documenting the class they should be in the class docstring instead. You can look at [this example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)